app/apps/addresses/views.py

```diff
@@ -33,6 +33,19 @@ def fetch_housing_corporations(auth_header=None):
     return response.json().get("results", [])
 
 
+def fetch_districts(auth_header=None):
+    url = f"{settings.ZAKEN_API_URL}/addresses/districts/"
+
+    response = requests.get(
+        url,
+        timeout=5,
+        headers=get_headers(auth_header),
+    )
+    response.raise_for_status()
+
+    return response.json().get("results", [])
+
```

Comment:
Dit is best wel hetzelfde als de functie daarboven (fetch_housing_corporations). Kan het misschien met een generieke helper functie wat DRY-er worden ?

Reply:
Zeker, is langzaam aan zo gegroeid, met ook meer varianten. Dit komt ook op meer plekken voor, dus misschien logisch om dit in een aparte pr op te pakken.

Reply:
Agreed